core/templates/pages/creator-dashboard-page/modal-templates/create-activity-modal.controller.spec.ts

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for CreateActivityModalController.
+ */
+
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// file is upgraded to Angular 8.
+import { UpgradedServices } from 'services/UpgradedServices';
+// ^^^ This block is to be removed.
+
+describe('Create Activity Modal Controller', function() {
+  var $q = null;
+  var $scope = null;
+  var $uibModalInstance = null;
+  var CollectionCreationService = null;
+  var ExplorationCreationService = null;
+  var UserService = null;
+
+  var userInfo = {
+    canCreateCollections: () => true
+  };
+
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    var ugs = new UpgradedServices();
```

Comment:
Should you be changing this based on Srijan's new changes?

Reply:
Sorry, which changes?